assets/components/contributionSelection/contributionSelectionReducer.js

```diff
@@ -0,0 +1,129 @@
+// @flow
```

Comment:
Is it possible to add tests for this Reducer?

Reply:
If this contributionSelection "presentational" component has an action and a reducer file associated with it, maybe it is better to create a folder to contain all these more complex components.

Reply:
Done.

To the second point, as discussed offline, we think this is a good idea. However, this is something we'd like to do to multiple components, so something for a future PR, when we have a good name.